pkg/schemas/database_parameters.go

```diff
@@ -0,0 +1,47 @@
+package schemas
+
+import (
+	"slices"
+	"strings"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var (
+	ShowDatabaseParametersSchema = make(map[string]*schema.Schema)
+	databaseParameters           = []sdk.AccountParameter{
+		sdk.AccountParameterDataRetentionTimeInDays,
+		sdk.AccountParameterMaxDataExtensionTimeInDays,
+		sdk.AccountParameterExternalVolume,
+		sdk.AccountParameterCatalog,
+		sdk.AccountParameterReplaceInvalidCharacters,
+		sdk.AccountParameterDefaultDDLCollation,
+		sdk.AccountParameterStorageSerializationPolicy,
+		sdk.AccountParameterLogLevel,
+		sdk.AccountParameterTraceLevel,
+		sdk.AccountParameterSuspendTaskAfterNumFailures,
+		sdk.AccountParameterTaskAutoRetryAttempts,
+		sdk.AccountParameterUserTaskManagedInitialWarehouseSize,
+		sdk.AccountParameterUserTaskTimeoutMs,
+		sdk.AccountParameterUserTaskMinimumTriggerIntervalInSeconds,
+		sdk.AccountParameterQuotedIdentifiersIgnoreCase,
+		sdk.AccountParameterEnableConsoleOutput,
+	}
+)
+
+func init() {
+	for _, param := range databaseParameters {
+		ShowDatabaseParametersSchema[strings.ToLower(string(param))] = ParameterSchema
+	}
+}
+
+func DatabaseParametersToSchema(parameters []*sdk.Parameter) map[string]any {
+	databaseParametersValue := make(map[string]any)
+	for _, param := range parameters {
+		if slices.Contains(databaseParameters, sdk.AccountParameter(param.Key)) {
```

Comment:
I don't like initializing `sdk.AccountParameter` like this, I like the warehouse impl more but it does not matter that much for now, we will soon generate them 🤷